### DISCUSSION TITLE
Cursor styling insert mode

### DISCUSSION THREAD

**Discussion by harg** [helix-editor/helix#11397] [Troubleshooting]
I've configured my cursor on insert mode like this :

```
[editor.cursor-shape] 
insert = "bar"
```

and in my theme set colors  for insert cursor : 

```
"ui.cursor.primary.insert" = { fg = "white", bg = "white" }
"ui.cursor.insert" = { fg = "white", bg = "white" }
```

Symptom: 
Theme colors configuration is ignored, the cursor take the color of the adjacent character.

Helix version : 24.07
Terminal : Allacritty, Wezterm, Windows Terminal, ...

**Comment by CptPotato** [comment]
Unfortunately, bar cursors cannot be themed afaik. The color is handled by the terminal emulator.

See also #8234, #10127, etc.
